com/win32comext/axscript/demos/client/ie/CHARTPY.HTM

```diff
@@ -20,7 +20,7 @@ Edited:	3/19/96 - VBScript code edited so there is only one procedure for each p
 <BODY BGCOLOR=#FFFFCC TEXT=#000000>
 
 <B><FONT SIZE=6>Chart Example</FONT></B><BR>
-<FONT SIZE=2>You must be running Microsoft Internet Explorer 3.0 and have the <A HREF ="http://microsoft.saltmine.com/isapi/activexisv/prmgallery/gallery-activex-info.idc?ID=162">Microsoft ActiveX Chart control</A> installed to view this page.</FONT><BR>
+<FONT SIZE=2>You must be running Microsoft Internet Explorer 3.0 and have the <A HREF ="http://microsoft.saltmine.com/isapi/activexisv/prmgallery/gallery-activex-info.idc?ID=162">Microsoft ActiveX Chart control (dead link)</A> installed to view this page.</FONT><BR>
```

Comment:
According to https://github.com/mhammond/pywin32/blob/034969e214e9353db1409b25c8ca61e1ddf8b4a1/com/win32com/readme.html#L62-L63
> Python only supports "trusted" execution hosts - thus, it will no longer work as an engine inside IE (Python itself no longer has a restricted execution environment).

This being a dead link probably doesn't matter much. In fact, IE Python ActiveX demos can probably be removed.

Reply:
yep, feel free to remove that.

Reply:
Keeping it scoped to only the link in this PR, but the entire file is removed in https://github.com/mhammond/pywin32/pull/2449